shell/artifacts/Social/PostTimeRanker.manifest

```diff
@@ -0,0 +1,13 @@
+// @license
+// Copyright (c) 2018 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+import 'Post.schema'
+
+particle PostTimeRanker in 'source/PostTimeRanker.js'
+  in [~a] input
```

Comment:
does PostTimeRanker work with anything else then Post?
The particle implementation expected the entities to have `createdTimestamp` field, so I think the particle spec should enforce at least that.

Reply:
Ah, thanks, done.